exporter/opentelemetry-exporter-otlp-proto-grpc/tests/metrics/test_otlp_metrics_exporter.py

```diff
@@ -89,6 +109,33 @@ def Export(self, request, context):
         return ExportMetricsServiceResponse()
 
 
+def _generate_metric(name, point) -> Metric:
```

Comment:
Single use function here

Reply:
This will be re-used when generating gauges and histograms as well.